packages/docs/scripts/generate-docs.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Requires jsdoc and jsdoc-to-markdown
+ * Reference: https://gist.github.com/slorber/0bf8c8c8001505f0f99a062ac55bf442
+ */
+
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+const jsdoc2md = require('jsdoc-to-markdown');
+const { execSync } = require('child_process');
+const sidebars = require('../sidebars');
+
+/**
+ * Runs through packages folders looking for JSDoc and generates markdown docs
+ */
+function generateDocs() {
+    console.log('-= Generating package docs =-');
+    // Use glob to get all js/ts files
+    const pathPattern = path.join(
+        __dirname,
+        '../../core/src/**/*.[jt]s?(x)'
+        // '../packages/**/src/**/*.[jt]s?(x)'
+    );
+    const filePaths = glob.sync(pathPattern, {
+        ignore: [
+            '**/node_modules/**',
+            '**/*.spec.js',
+            '**/*.d.ts',
+            '**/index.js',
+            '**/*.stories.js',
+            '**/*DataBuilder.js',
+            '**/*Data.js',
+        ],
```

Comment:
this could be a const outside the function.

Reply:
Will do